exercises/etl/.meta/template.j2

```diff
@@ -0,0 +1,20 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        legacy_data =  { {% for key, value in case["input"]["legacy"].items() %}
+                            {{key}}: {{value}}{{ "," if not loop.last else "" }}{% endfor %} }
```

Comment:
Since `black` is going to add the comma anyway, might I suggest simplifying the template?

```suggestion
                            {{key}}: {{value}},{% endfor %} }
```